themes/default/layouts/page/reinvent.html

```diff
@@ -0,0 +1,125 @@
+{{ define "hero" }}
+    {{ partial "hero.html" (dict "title" .Params.title) }}
+{{ end }}
+
+{{ define "main" }}
+    <section id="ctas" class="container mx-auto my-16 text-center">
+        <h2>Meet with us at Booth 739!</h2>
+
+        <div class="my-16">
+            <div class="lg:flex lg:flex-wrap justify-center items-stretch">
+                <!--<div class="lg:w-1/2 px-3 py-6">
+                    <div class="card text-center p-6 relative h-full">
+                        <div class="icon-section">
+                            {{ partial "color-icon.html" (dict "icon" "code" "icon_color" "blue") }}
+                        </div>
+                        <div>
+                            <h5>Announcing Pulumi Crosswalk for AWS v1.0</h5>
+                        </div>
+                        <div class="description">
+                            <p>Explore our multi-language library of well-architected AWS infrastructure patterns for rapid cloud development. Check it out on <a href="https://github.com/pulumi/pulumi-awsx" target="_blank">GitHub <i class="fab fa-github"></i></a></p>
+                        </div>
+                        <div class="card-cta-btn">
+                            <a href="{{ relref . "/blog/introducing-pulumi-crosswalk-for-aws-the-easiest-way-to-aws" }}" class="btn-primary">Read the Blog</a>
+                        </div>
+                    </div>
+                </div>-->
```

Comment:
just in case you didn't mean to leave this commented out code in.

Reply:
Yep! Gonna get uncommented with the correct blog link once it's published (supposed to be 11/28)